src/main/sensors/compass.c

```diff
@@ -267,6 +268,16 @@ bool compassDetect(magDev_t *dev, uint8_t *alignment)
         break;
     }
 
+#ifdef USE_MAG_MPU925X_AK8963
```

Comment:
Would be helpful to add a comment here about how this device is being treated differently. As it it's a MPU9250 that's not being used for gyro/acc but instead configured as a passthrough to the embedded AK8963 only. This way those that come after you will understand the reason for the logic.